server/.gitignore

```diff
@@ -10,7 +10,7 @@ npm-debug.log
 webapp/yarn-error.log
 mattermost.mattermost-license
 config/mattermost.mattermost-license
-config/config.json
+config/*.json
```

Comment:
0/5 How about `config*.json` to keep things within scope?

Reply:
Even better